### ISSUE TITLE
files/etc/tmpfiles.d/sap.conf is partially already included in systemd package

### ISSUE THREAD

**Issue by Klaas-** [linux-system-roles/sap-preconfigure#28]
files/etc/tmpfiles.d/sap.conf:
```
# systemd.tmpfiles exclude file for SAP
# SAP software stores some important files in /tmp which should not be deleted automatically.
# This file has been created using role sap-preconfigure in RHEL System Roles for SAP.
# Do not change this file as it might be overwritten when running role sap-preconfigure again.
 
# Exclude SAP socket and lock files
x /tmp/.sap*
 
# Exclude HANA lock file
x /tmp/.hdb*lock

# Exclude TREX lock file
x /tmp/.trex*lock
```

systemd-219-62.el7_6.9.x86_64 (/usr/lib/tmpfiles.d/sap.conf):
```
# systemd tmpfiles exclude file for SAP
# SAP software unfortunately stores some important files
# in /tmp which should not be deleted

# Exclude SAP socket and lock files
x /tmp/.sap*

# Exclude HANA lock file
x /tmp/.hdb*lock
```

the first two excludes are already included in rhel 7.6 systemd. Apparently it has been in systemd since 7.2 https://bugzilla.redhat.com/show_bug.cgi?id=1186044 if I am not mistaken :)

**Comment by berndfinger** [comment]
Many thanks for this. Unfortunately, /usr/lib/tmpfiles.d/sap.conf disappeared from RHEL 8 onwards. Maybe the role should check for the file and copy the file in the role if it does not exist, and add the TREX requirement if the file is there.

**Comment by berndfinger** [comment]
As a first approach, the role will check if /usr/lib/tmpfiles.d/sap.conf exists and if yes, /etc/tmpfiles.d/sap.conf will not be copied.

**Comment by berndfinger** [comment]
fixed by pull request #56.
Thanks for your inputs Klaas- and rhmk!

**Comment by Klaas-** [comment]
My general point was more "should this be in systemd or should this be part of sap configuration and why are those configurations different" :) but the solution is fine for me. From my point of view this should not be part of systemd because it's an application configuration.
